jans-cedarling/cedarling/src/bootstrap_config/decode.rs

```diff
@@ -408,19 +407,22 @@ impl BootstrapConfig {
 
         // Decode policy store
         let policy_store_config = match (
+            raw.local_policy_store.clone(),
             raw.policy_store_uri.clone(),
             raw.policy_store_local_fn.clone(),
         ) {
             // Case: no policy store provided
-            (None, None) => Err(BootstrapDecodingError::MissingPolicyStore)?,
-
+            (None, None, None) => Err(BootstrapDecodingError::MissingPolicyStore)?,
```

Comment:
We no need match statement here

How about if else statement?

Reply:
It's better to use match here since we're not only checking if one policy store exists then loading the first one we see -- we also need to return an error if multiple policy stores were given.

Reply:
I don`t agree, but nevermind